pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1035,7 +1034,6 @@ public void maxpeersOptionMustBeUsed() {
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
-  @Ignore("Ignored as we only have one mode available for now. See NC-1057/NC-1681")
   @Test
   public void syncModeOptionMustBeUsed() {
```

Comment:
Also 2 new tests similar to `syncModeOptionMustBeUsed` to check the two new options values are well passed from the CLI to the SynchronizerConfiguration would help make sure we don't have an issue with a valid integer passed but not matching the entered value on the command line.

Reply:
tests added